packages/server/src/internalPlugin.ts

```diff
@@ -30,7 +30,8 @@ export type InternalPluginId =
   | 'LandingPageDisabled'
   | 'SchemaReporting'
   | 'InlineTrace'
-  | 'UsageReporting';
+  | 'UsageReporting'
+  | 'SubscriptionCallback';
```

Comment:
IIRC the point of "internal plugins" is to allow us to install them by default (and detect if they've already been installed). If there's no default-installation behavior we don't need this (eg, the HTTP drain plugin isn't here).

Reply:
[ee1c3ab](https://github.com/apollographql/apollo-server/pull/7617/commits/ee1c3ab4a058ce2a0bbfe0d1bdaf33cd49387beb)